oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -154,6 +154,13 @@ public class ServiceAccountCredentials extends GoogleCredentials
    * Returns service account credentials defined by JSON using the format supported by the Google
    * Developers Console.
    *
+   * <p>Important: If you accept a credential configuration (credential JSON/File/Stream) from an
+   * external source for authentication to Google Cloud Platform, you must validate it before
+   * providing it to any Google API or library. Providing an unvalidated credential configuration to
+   * Google APIs can compromise the security of your systems and data. For more information, refer
+   * to {@link <a
+   * href="https://cloud.google.com/docs/authentication/external/externally-sourced-credentials">documentation</a>}.
+   *
```

Comment:
Just noticed this method is package-private. I think we do not publish javadoc for this.
https://github.com/googleapis/google-auth-library-java/blob/d81aafeeeaf39cf69000596f1f2b0f2fc8cc2141/oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java#L163

Reply:
Updated to reference `fromStream()` instead